crypto4pl.c

```diff
@@ -66,6 +68,255 @@ typedef enum
 } crypt_mode_t;
 
 
+                 /***************************
+                 *         HASHING          *
+                 ****************************/
+
+
+#define CONTEXT_MAGIC (~ 0x53481284L)
+
+typedef struct context
+{ int             magic;
+  atom_t          atom;
```

Comment:
I used `ssl4pl.c` as an implementation example. To me, this member is one of the few that made immediate sense when I saw the library: It is the atom registered for this context. I cannot think of a better name to denote an atom. However, if you want to change it, please also change it in the other libraries where this technique is used.